RoddyCore/src/de/dkfz/roddy/config/loader/ConfigurationFactory.groovy

```diff
@@ -148,6 +148,8 @@ class ConfigurationFactory {
                         availableConfigurations[iccSub.id] = iccSub
                     }
 
+                } catch (UnknownConfigurationFileTypeException ex) {
```

Comment:
Hm, not sure whether every individual error needs its own type. I have no experience with this but will this not result in an explosion of exception types? Are there any standards for when to make it a type and when to just write it down in the message? e.g. make it a type if it has to be discriminated from other such (configuration) exceptions in a catch clause?